cmd/root.go

```diff
@@ -32,13 +33,29 @@ import (
 
 var configOptions *ConfigOptions
 
+// Start setups and starts the CLI.
+func Start() {
+	// flags := pflag.NewFlagSet("falcoctl", pflag.ExitOnError)
+	// pflag.CommandLine = flags
+	root := New()
+	if err := root.Execute(); err != nil {
+		logger.Critical("error executing falcoctl: %s", err)
+		os.Exit(1)
+	}
+}
+
 func init() {
 	configOptions = NewConfigOptions()
 	cobra.OnInitialize(initConfig)
 }
 
+// FalcoctlRoot wraps the root cobra.Command.
+type FalcoctlRoot struct {
```

Comment:
I believe we can avoid this wrapper, because the same result can be achieved using `cobra.Command` directly.
[Here](https://github.com/leogr/event-generator/blob/master/cmd/root.go) an example.